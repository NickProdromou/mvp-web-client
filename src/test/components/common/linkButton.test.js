import React from 'react';
import LinkButton from '../../../components/common/linkButton/linkButton';
import renderer from 'react-test-renderer';

it('renders correctly', () => {

    const tree = renderer.create(
        <LinkButton
            to="/a-link"
            text="Clickonme"
        />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
