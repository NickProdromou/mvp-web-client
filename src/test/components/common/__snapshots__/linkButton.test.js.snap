// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<a
  className="sc-bdVaJa bRdZkh"
  href="/a-link"
>
  Clickonme
</a>
`;
